docs/reference/linstorsatelliteconfiguration.md

```diff
@@ -78,8 +87,11 @@ This examples sets three Properties on every satellite:
 * `AutoplaceTarget` (if set to `no`, will exclude the node from LINSTOR's Autoplacer) takes the value from the
   `piraeus.io/autoplace` annotation of the Kubernetes Node. If a node has no `piraeus.io/autoplace` annotation, the
   property will not be set.
-* `Aux/role/$1` copies all "node-role.kubernetes.io/*" label keys and values. For example, a worker node with the
-  `node-role.kubernetes.io/worker: "true"` label will have the `Aux/role/worker` set to `"true"`.
+* `Aux/role/` copies all "node-role.kubernetes.io/*" label keys and values. For example, a worker node with the
```

Comment:
Strictly we should probably format all these example properties as `node-role.kubernetes.io/*` instead of "node-role.kubernetes.io/*". Might end up being too much code style, though :smile: